metals/src/main/scala/scala/meta/internal/metals/JavaTarget.scala

```diff
@@ -0,0 +1,23 @@
+package scala.meta.internal.metals
+
+import java.{util => ju}
+
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.io.AbsolutePath
+
+import ch.epfl.scala.bsp4j.JavacOptionsItem
+
+trait JavaTarget extends CommonTarget {
+
+  val javac: JavacOptionsItem
+
+  def isJavaSemanticdbEnabled: Boolean = javac.isSemanticdbEnabled
```

Comment:
```suggestion
  def semanticdbEnabled: Boolean = javac.isSemanticdbEnabled
```
if we put it in the main trait, we could just return `Target` or `CommonTarget`  and maybe only check a method `isJavaOnlyTarget` when needed.